nominal:
  goal:
    coordinates:
      - 0.0
      - 2.2
      - 1.3
      - 0.0
      - 0.0
      - 0.0
      - 0.0
    gain:
    - [0.0, 0.2, 0.0, 0.0, 0.2, 0.0, 0.0]
    - [0.2, 0.0, 0.0, 0.2, 0.0, 0.0, 0.0]
    - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
    - [0.0, 0.0, -5.4772, 0.0, 0.0, -5.5637, 0.0]
    u_ref:
      - 0.0
      - 0.0
      - 0.0
      - 10.5
  frequency: 10.0
cbf:
  gamma: 0.2
  slack: True

limits:
  max_thrust: 62000
  min_thrust: 16000
  max_roll: 30.0
  max_pitch: 30.0
  max_yawrate: 30.0